packages/app/src/vis-packs/core/heatmap/MappedHeatmapVis.tsx

```diff
@@ -19,7 +20,7 @@ import { useSafeDomain, useVisDomain } from './hooks';
 interface Props {
   dataset: Dataset<ArrayShape, NumericType>;
   selection: string | undefined;
-  value: number[];
+  value: number[] | TextureTypedArray;
```

Comment:
The key change for this PR.

Note that I renamed `CompatibleTypedArray` to `TextureTypedArray`, as "compatible" didn't convey much about what the type was compatible with when used outside of `HeatmapMesh` 😅 -- I'm open to other naming suggestions, of course. I hesitated with `HeatmapTypedArray` but thought it was worth clarifying that the restriction came from the ThreeJS texture.

Reply:
`TextureTypedArray` is great :+1: